docs/commerce/payment/create_payment_methods.md

```diff
@@ -24,15 +27,26 @@ With the `Payment method/Edit` permission, you can modify existing ones.
 
 ![Creating a new payment method](create_new_payment_method.png)
 
-4\. Click **Create** to save your changes.
+4\. Toggle the **Availability** switch on, so that store customers can select this shipping method during checkout.
+
+5\. Click **Create** to save your changes.
 
 ## Edit an existing payment method
 
-1. In the left panel, go to **Commerce** -> **Payment methods**.
+1\. In the left panel, go to **Commerce** -> **Payment methods**.
 
 ![Payment methods list](payment_methods_list.png)
 
-2. Find the payment method that you intend to edit by using the search field and filters.
-3. Click the **Edit** button next to the method in the list.
-4. Edit the necessary details.
-5. Click **Update** to save your changes.
+2\. Find the payment method that you intend to edit by using the search field and filters.
+
+3\. If you are deleting a shipping method, select a box next to its name and click **Delete**, which concludes the procedure. Otherwise, skip to step 4.
```

Comment:
I'd move this to a separate section, otherwise a note. The procedure is "edit payment method", so mentioning deleting is interrupting the flow.